### ISSUE TITLE
Warnings from ruby 2.7

### ISSUE THREAD

**Issue by iay** [nanoc/nanoc#1479]
### Steps to reproduce

1. Run nanoc on a site using ruby 2.7, released December 2019

### Expected behavior

No change from Ruby 2.6.

### Actual behavior

Deprecation warnings from Ruby 2.7, most about other packages but a couple about nanoc itself. 

This is far from a serious problem (I don't even plan to migrate to 2.7 soon) but I thought putting the issues in record would be useful.

### Details

```
/Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/cri-2.15.9/lib/cri/command_dsl.rb:243: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/cri-2.15.9/lib/cri/command_dsl.rb:155: warning: The called method `option' is defined here
/Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/cri-2.15.9/lib/cri/command_dsl.rb:220: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/cri-2.15.9/lib/cri/command_dsl.rb:155: warning: The called method `option' is defined here
/Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/json-2.1.0/lib/json/common.rb:156: warning: Using the last argument as keyword parameters is deprecated
/Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/json-2.1.0/lib/json/common.rb:156: warning: Using the last argument as keyword parameters is deprecated
Loading site… /Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/json-2.1.0/lib/json/common.rb:156: warning: Using the last argument as keyword parameters is deprecated
/Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/json-2.1.0/lib/json/common.rb:156: warning: Using the last argument as keyword parameters is deprecated
done
/Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/nanoc-core-4.11.14/lib/nanoc/core/compiler_loader.rb:35: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/Volumes/Cased/git/iay/iay/site/vendor/ruby/2.7.0/gems/nanoc-core-4.11.14/lib/nanoc/core/compiler.rb:8: warning: The called method `initialize' is defined here
```

**Comment by joerggollnick** [comment]
I'm working already with Ruby 2.7.

Latest release for cri removed related warnings.

In my installation these 2 nanoc related are remaining.

/usr/lib/ruby/gems/2.7.0/gems/nanoc-core-4.11.14/lib/nanoc/core/compiler_loader.rb:35: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/lib/ruby/gems/2.7.0/gems/nanoc-core-4.11.14/lib/nanoc/core/compiler.rb:8: warning: The called method `initialize' is defined here

**Comment by ddfreyne** [comment]
The ones for Cri are fixed, but the ones for Nanoc are still pending — they are fixed on `master`, but not yet released.

**Comment by ddfreyne** [comment]
A release is coming up soon.

**Comment by iay** [comment]
Was this supposed to be fixed in 4.11.15? I am (I think) still seeing some of these:

```
/builds/iay/site/vendor/ruby/2.7.0/gems/nanoc-core-4.11.15/lib/nanoc/core/compiler_loader.rb:35: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/builds/iay/site/vendor/ruby/2.7.0/gems/nanoc-core-4.11.15/lib/nanoc/core/compiler.rb:8: warning: The called method `initialize' is defined here
```

**Comment by ddfreyne** [comment]
@iay There were indeed still some issues — a fix is in #1498.
